### PR TITLE
fix(docs): Improve the documentation for the new release

### LINKED ISSUE #626: fix(docs): Improve documentation: improve contributor guide
In https://github.com/probabl-ai/skore/pull/618, I took the old [CONTRIBUTING.md](https://github.com/probabl-ai/skore/blob/main/CONTRIBUTING.md) and made a few changes to the contributor guide on the documentation. Maybe we should try updating the and giving more details.

Ideas of improvements:
- [ ] include Auguste's remarks in https://github.com/probabl-ai/skore/pull/541#pullrequestreview-2382199356
- [ ] do a specific subpage for documentation

### PR DESCRIPTION
**Pull request by sylvaincom**
Fixing parts of https://github.com/probabl-ai/skore/issues/627 
- removed getting started and user guide tabs as they are in examples
- added a convincing getting started example following @koaning advice (thanks again)
- merged the two examples on the skore UI into one
- In `CONTRIBUTING.md`, made a link to the documentation
- Updated the GIF in the README and in the documentation
- In the cross-validation example, no longer use Auguste's screenshot but in the notebook I export a png that is then loaded
    - added `kaleido` to `pyproject.toml`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,91 +0,0 @@
-# Contributing guide
```
Comment (thomass-dev) [inline CONTRIBUTING.md:1]: Why this file was removed?

**Review by thomass-dev — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (thomass-dev) [reply CONTRIBUTING.md:1]: Do you think you can merge `sphinx/contributor_guide.rst` in `CONTRIBUTING.md`, and use this file in sphinx?

Comment (augustebaum) [reply CONTRIBUTING.md:1]: in sklearn CONTRIBUTING.md links to the docs. this is not a bad thing to me.

Comment (sylvaincom) [reply CONTRIBUTING.md:1]: My previous comment that I don't know if got sent:
> I put back the `CONTRIBUTING.md` file and in it I added a link to the documentation

Comment (sylvaincom) [reply CONTRIBUTING.md:1]: I'm looking into this https://github.com/sphinx-doc/sphinx/issues/7000

Comment (sylvaincom) [reply CONTRIBUTING.md:1]: Done

Comment (thomass-dev) [reply CONTRIBUTING.md:1]: Nice, well done!

**Review by thomass-dev — DISMISSED** [review]

File: CONTRIBUTING.md
Comment (augustebaum) [inline CONTRIBUTING.md:1]: I guess that also works, but I expected the CONTRIBUTING.md to just be shortened to something like:
```
To contribute, take a look at the docs at <http://skore/docs/contributing>
```
, not to be a mirror of the online docs

File: examples/plot_01_getting_started.py [hunk 2]
```diff
@@ -1,125 +1,178 @@
...
-my_pipeline = Pipeline(
-    [("standard_scaler", StandardScaler()), ("lasso", Lasso(alpha=2))]
-)
-my_pipeline.fit(X, y)
+# %%
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
 
-my_project_gs.put("my_fitted_pipeline", my_pipeline)
+fig_plotly.write_image("plot_01_cross_validation.png", scale=2)
```
Comment (augustebaum) [inline examples/plot_01_getting_started.py:153]: That's great, I'm glad it works.

**Review by augustebaum — APPROVED** [review]

File: CONTRIBUTING.md
Comment (sylvaincom) [reply CONTRIBUTING.md:1]: Created an issue for that: https://github.com/probabl-ai/skore/issues/642

**Pull request merged**